gestore/management/commands/importobjects.py

```diff
@@ -0,0 +1,336 @@
+from typing import List, Tuple
+
+from django.core.management import CommandError
+from django.core.serializers.python import Deserializer
+from django.core.management.color import no_style
+from django.db import (
+    connections,
+    DatabaseError,
+    DEFAULT_DB_ALIAS,
+    IntegrityError,
+    router,
+    transaction,
+)
+
+from gestore.gestore_command import GestoreCommand
+from gestore.typing import PK
+from gestore.utils import (
+    write_packages_diff,
+    get_pip_packages,
+    get_str_from_model,
+    has_conflict
+)
+
+
+class Command(GestoreCommand):
+    """
+    Imports objects from an export file.
+    """
+    def __init__(self, *args, **kwargs):
+        self.export_object_count = 0
+        self.loaded_object_count = 0
+        self.to_save_objects = []
+        self.using = DEFAULT_DB_ALIAS
+        self.ignore = False
+        self.override = False
+
+        super(Command, self).__init__(*args, **kwargs)
+
+    def add_arguments(self, parser) -> None:
+        # Add common args
+        super(Command, self).add_arguments(parser)
+
+        parser.add_argument(
+            'path',
+            help='The path of the exports file',
+            type=str,
+        )
+        parser.add_argument(
+            '-o', '--override',
+            action='store_true',
+            default=False,
+            help='Override conflicts in DB. This is very dangerous, please '
+                 'use with care'
+        )
```

Comment:
@ahmedaljazzar What user stories this option enables? There's a small chance that this command would be misused and creates confusion and a need for manual data recovery.

Please either remove this option, or make it work in devstack or other `settings.DEBUG` mode.


```suggestion
```